src/west/_bootstrap/main.py

```diff
@@ -259,36 +270,25 @@ def reinit(config_path, args):
     This updates the west/config configuration file, and optionally resets the
     manifest, west, and project repositories to the new revision.
     '''
-    if args.base_url:
-        if args.west_url or args.manifest_url:
-            sys.exit('fatal error: -b is incompatible with -m and -w')
-        west_url = args.base_url.rstrip('/') + '/west'
-        manifest_url = args.base_url.rstrip('/') + '/manifest'
-    else:
-        west_url = args.west_url
-        manifest_url = args.manifest_url
+    manifest_url = args.manifest_url
 
-    if not (west_url or args.west_rev or manifest_url or args.manifest_rev):
+    if not (manifest_url or args.manifest_rev):
         sys.exit('West already initialized. Please pass any settings you '
                  'want to change.')
 
-    update_conf(config_path, west_url, args.west_rev, manifest_url,
-                args.manifest_rev)
+    update_conf(config_path, manifest_url, args.manifest_rev)
 
     print('=== Updated configuration written to {} ==='.format(config_path))
 
     if args.reset:
         cmd = ['update']
-        if manifest_url or args.manifest_rev:
-            cmd += ['--reset-manifest', '--reset-projects']
-        if west_url or args.west_rev:
-            cmd.append('--reset-west')
+        cmd += ['--reset-manifest', '--reset-projects', '--reset-west']
```

Comment:
Could just have

```python
cmd = ['update', '--reset-manifest', '--reset-projects', '--reset-west']
```

since the command is fixed now.

Reply:
Yep, good point. Fixed.